plugins/cloudconfigs/azure/plugin/azure.go

```diff
@@ -0,0 +1,94 @@
+package plugin
+
+import (
+	"fmt"
+
+	"github.com/enaml-ops/enaml"
+	"github.com/enaml-ops/omg-cli/plugins/cloudconfigs"
+	"github.com/enaml-ops/omg-cli/plugins/cloudconfigs/azure/plugin/generated"
+	"gopkg.in/urfave/cli.v2"
+	"gopkg.in/yaml.v2"
+)
+
+type NetworkCloudProperties struct {
```

Comment:
I would make `NetworkCloudProperties` and `AzureCloudConfig` unexported (ie lower case)